eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/boot/AbstractHTTPServer.java

```diff
@@ -121,7 +124,7 @@ public abstract class AbstractHTTPServer extends AbstractRemotingServer {
     private final transient EventMeshHTTPConfiguration eventMeshHttpConfiguration;
 
     private final transient ThreadPoolExecutor asyncContextCompleteHandler =
-            ThreadPoolFactory.createThreadPoolExecutor(10, 10, "EventMesh-http-asyncContext");
+        ThreadPoolFactory.createThreadPoolExecutor(10, 10, "EventMesh-http-asyncContext");
```

Comment:
Would you like to open an issue to modify the code style?

Reply:
I noticed that there are a lot of style modifications.

Reply:
> Would you like to open an issue to modify the code style?

@mytang0  I will create issue to modify the code style base on eventmesh-code-style.xml

Reply:
Thanks, great.